packages/skills/weather_client_ledger/handlers.py

```diff
@@ -333,18 +320,18 @@ def handle(self, message: Message, sender: str) -> None:
             fipa_msg = cast(FIPAMessage, dialogue.last_incoming_message)
             new_message_id = cast(int, fipa_msg.get("message_id")) + 1
             new_target_id = cast(int, fipa_msg.get("target")) + 1
-            counterparty_pbk = dialogue.dialogue_label.dialogue_opponent_pbk
             inform_msg = FIPAMessage(message_id=new_message_id,
                                      dialogue_reference=dialogue.dialogue_label.dialogue_reference,
                                      target=new_target_id,
                                      performative=FIPAMessage.Performative.INFORM,
                                      info=json_data)
             dialogue.outgoing_extend(inform_msg)
-            self.context.outbox.put_message(to=counterparty_pbk,
+            self.context.outbox.put_message(to=message.counterparty,
```

Comment:
please revert this, the counterparty of the message is the decision maker..not what we want...so this needs to be as it was